qemu/qemu_test.go

```diff
@@ -980,3 +981,21 @@ func TestBadCPUs(t *testing.T) {
 		t.Errorf("Error expected")
 	}
 }
+
+// -pci devices don't play well with Z hence replace them with corresponding -ccw devices
+// See https://wiki.qemu.org/Documentation/Platforms/S390X
+func init() {
```

Comment:
you can create a new file called `qemu_s390x_test.go` and move this function there

Reply:
done